### ISSUE TITLE
Poor result

### ISSUE THREAD

**Issue by Alexander4202333** [ZhaochongAn/COSeg#19]
![Image](https://github.com/user-attachments/assets/876856fe-42b6-4e07-855a-f26ea5ab335e)

![Image](https://github.com/user-attachments/assets/b533522b-09d4-42d6-8d6b-1c5f577aadba)

Hello, I tried downloading the dataset, corresponding pre-trained models, and the provided trained model. However, the final test results seem to be very poor. What could possibly be the issue?

**Comment by ZhaochongAn** [comment]
Hi, could you please check whether your generated eval data set is empty or not? It seems your evaluation data folder may not be correctly created. The evaluation set is generated on-the-fly during the first run. In subsequent runs, the script simply detects the existence of the evaluation folder and loads the saved data inside (see [this code](https://github.com/ZhaochongAn/COSeg/blob/2811279d686f15256393eb4188ad042c4fb93b14/util/s3dis_fs.py#L695)).

If your first run crashed or was interrupted during test data creation, you will end up with an incomplete eval set. Any following runs would continue to use this partial data. Please verify whether the test set was fully created. If not, delete the eval data folder and re-run the code to ensure the test data is fully saved.

**Comment by Alexander4202333** [comment]
![Image](https://github.com/user-attachments/assets/26c48f56-b6e0-4c19-b275-68f43d3592d2)
Hello, I tried deleting the generated data and regenerating it for inference, and the results have improved somewhat but still seem insufficient. What could be the possible issue? Are there any other places in the YAML file besides data_root, pretrain_backbone, and save_path that need to be modified?

**Comment by ZhaochongAn** [comment]
Hi, thanks for the update. In the config file, you should make sure that `cvfold`, `n_way`, `k_shot` align with the used model weight. For example, if you are using model "s31_1w1s" you should set cvfold = 1, n_way = 1, and k_shot = 1. Could you verify that these values are correctly configured?
